contracts/lqmining/simple/LMPoolSelfHosted.sc

```diff
@@ -113,7 +113,7 @@
   val deltaTMP = poolTMP1._2 - poolTMP0._2
 
   // ===== Calculating epoch parameters ===== //
-  val epochAlloc    = programBudget0 / epochNum
+  val epochAlloc    = (programBudget0 - 1L) / epochNum
```

Comment:
Why? We can set correct program budget in the register. Then initial `tokens(1)._1` will be `programBudget0 + 1`

Reply:
Fixed it in the last commit